libs/content/workspace/src/background.rs

```diff
@@ -33,16 +30,13 @@ struct WorkerState {
     last_auto_save: Instant,
     last_auto_sync: Instant,
     last_sync_stat: Instant,
-
-    user_last_seen: Instant,
```

Comment:
removed - it was broken, it doesn't need to be fixed right now, and it was making the sync frequency effectively 1 hour

Reply:
update: new mechanism for this in workspace.rs